client/src/js/components/bhExchangeRate.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @overview bhExchangeRate
+ *
+ * @description
+ * This component allows to display the exchange rate
+ */
+angular.module('bhima.components')
+  .component('bhExchangeRate', {
+    templateUrl : 'modules/templates/bhExchangeRate.tmpl.html',
+    controller : bhExchangeRateController,
+    controllerAs : '$ctrl',
+  });
+
+bhExchangeRateController.$inject = [
+  'CurrencyService', 'ExchangeRateService', 'SessionService',
+  'NotifyService', 'moment',
+];
+
+function bhExchangeRateController(Currencies, Rates, Session, Notify, Moment) {
+  var $ctrl = this;
+
+  $ctrl.today = new Date();
+  $ctrl.primaryExchange = {};
+  $ctrl.enterprise = Session.enterprise;
+  $ctrl.EXCHANGE_RATE_DISPLAY_SIZE = 6;
+
+  /*
+    $ctrl.isFirstCurencyLabel is used to check the exchange Rate
+    is lower then 1  the program display something
+    mutch better for reading
+  */
+  $ctrl.isFirstCurencyLabel = false;
+
+  // load exchange rates
+  function loadExchangeRates() {
+    Currencies.read(true)
+      .then((currencies) => {
+        $ctrl.currencies = currencies.filter((currency) => {
+          return currency.id !== Session.enterprise.currency_id;
+        });
+
+        // load supported rates
+        return Rates.read(true);
+      })
+      .then(() => {
+        $ctrl.currencies.forEach((currency) => {
+          currency.rate = Rates.getCurrentRate(currency.id);
```

Comment:
Is this the same code used on the home page?  If so, we should probably put it in a service ... something like `Exchange.getHumanReadableRate()`.